source/operations/monitoring/metrics-and-alerts.rst

```diff
@@ -613,6 +613,26 @@ Lock Metrics
 
    Number of current READ locks on the peer.
 
+
+Webhook Metrics
+~~~~~~~~~~~~~~~
+
+.. metric:: minio_cluster_webhook_failed_messages
+
+   Number of messages that failed to send.
+
+.. metric:: minio_cluster_webhook_online
+
+   State of the webhook endpoint: `1` if online, or `0` if not online.
```

Comment:
It doesn't appear we explicitly say what type of metric something is. This one is an instantaneous measurement (gauge) rather than the more common counter. Should we go into that? Or leave it at the descriptions.

Reply:
For now at least we can leave it - if we had a programmatic way to populate these I would be more interested in that level of detail, but as-is if something changes we can't easily catch it.

The metrics themselves when pulled via CURL do state what they are, so I assume users can figure it out from there.

Reply:
We could try to pull it from the code, I suppose. Each metric has the Type defined in https://github.com/minio/minio/blob/master/cmd/metrics-v2.go, but I wouldn't even know where to begin to try to automate that.

(For example, https://github.com/minio/minio/blob/0474791cf8b990c7d44774e7f06f55c415dc3291/cmd/metrics-v2.go#L2399 defines it for the Webhook Online metric.)

Reply:
Yeah looks messy to extract details programmatically, certainly not an `awk` operation. `Help` is conveniently a string, but the rest are const variables. The complete metric names don't exist in the code from what I can see.